tripal_ws/includes/TripalWebService/TripalContentService_v0_1.inc

```diff
@@ -371,8 +371,21 @@ class TripalContentService_v0_1 extends TripalWebService {
       }
     }
 
-    if (isset($field['field_permissions']['type']) && $field['field_permissions']['type'] != 0) {
-      return;
+    if (isset($field['field_permissions']['type'])) {
+
+      global $user;
+
+      if ($field['field_permissions']['type'] === '1') {
```

Comment:
Just curious, where do these 'types' (e.g. 1 and 2) come from? Is this a field_permission module thing?

Reply:
yep!  we could link out to the field permissions module in a comment if that would help although i only found it in the code, not in the online documentation.

its 0 for no permissions, 1 for private, 2 for custom.

although that makes me realize, maybe the action for 1 & 2 should be the same- to check the user's permission.

Reply:
Are you content @bradfordcondon or did you want to change this before merging? I feel your original approach accomplishes the same goal.

Reply:
thanks for the review, and the reminder about this @laceysanderson 

Basically the difference is that if the admin user is using web services, or if the user is the author, they wont be able to access the field via web services.

It can be easily resolved by taking the same approach for permissions 1 & 2 :

```php

   if (!user_access('view ' . $field_name, $user)) {
          return;
        }

```